.readthedocs.yml

```diff
@@ -0,0 +1,14 @@
+version: "2"
+
+build:
+  os: "ubuntu-24.04"
+  tools:
+    python: "3.13"
+
+  jobs:
+    post_install:
+      - pip install poetry
+      - VIRTUAL_ENV=$READTHEDOCS_VIRTUALENV_PATH poetry install --with docs
+
+  mkdocs:
+    configuration: mkdocs.yml
```

Comment:
🤦 This key is indented too far according to the [relevant schema used by this file](https://github.com/readthedocs/readthedocs.org/blob/1d50bb7dbf3d9bd9efd878dea93e1373033a7f50/readthedocs/rtd_tests/fixtures/spec/v2/schema.json).